query-connector/src/app/api/query/route.ts

```diff
@@ -16,10 +16,8 @@ import {
 } from "../../constants";
 
 import { handleRequestError } from "./error-handling-service";
-import {
-  getSavedQueryByName,
-  mapQueryRowsToValueSets,
-} from "@/app/database-service";
+import { getSavedQueryByName } from "@/app/database-service";
+import { mapQueryRowsToValueSets } from "@/app/utils";
```

Comment:
moving this per @fzhao99's suggestion [here](https://github.com/CDCgov/dibbs-query-connector/pull/184#discussion_r1868921735)